packages/react/src/BacktraceClient.ts

```diff
@@ -0,0 +1,37 @@
+import {
+    BacktraceAttributeProvider,
+    BacktraceCoreClient,
+    BacktraceRequestHandler,
+    BacktraceStackTraceConverter,
+} from '@backtrace/sdk-core';
+import { AGENT } from '@backtrace/browser';
+import { BacktraceConfiguration } from '@backtrace/browser';
+import { BacktraceClientBuilder } from '@backtrace/browser';
+
+export class BacktraceClient extends BacktraceCoreClient {
+    private static _instance?: BacktraceClient;
+    constructor(
+        options: BacktraceConfiguration,
+        handler: BacktraceRequestHandler,
+        attributeProviders: BacktraceAttributeProvider[],
+        stackTraceConverter: BacktraceStackTraceConverter,
+    ) {
+        super(options, AGENT, handler, attributeProviders, stackTraceConverter);
+    }
+
+    public static builder(options: BacktraceConfiguration): BacktraceClientBuilder {
+        return new BacktraceClientBuilder(options);
+    }
+
+    public static initialize(options: BacktraceConfiguration): BacktraceClient {
```

Comment:
Why do we need to use the initialize method? Could you take a look at you're calling builder to build the class and store it in the class instance of the client? You can do the same in the constructor, which is a more intuitive way for me. 

WIth this approach, you have one way to initialize everything. Right now, we have a different way to initialize other clients and we don't know which one (builder vs client) is preferred.

Reply:
I propose we add an initialize method to all of our clients that calls the builder. I think it would provide a good user experience as the user would not be required to know implementation details about which patterns we use to create the client.

Reply:
In some way I agree with @adamcronin42 that `.initialize` may be a better method. Or the one that actually initializes Backtrace in a global context. `.build` could only create the instance, and not assign it as a global error handler.